pilot/pkg/features/pilot.go

```diff
@@ -300,7 +300,7 @@ var (
 		return durationpb.New(defaultRequestTimeoutVar.Get())
 	}()
 
-	EnableServiceApis = env.RegisterBoolVar("PILOT_ENABLED_SERVICE_APIS", true,
+	EnableGatewayAPI = env.RegisterBoolVar("PILOT_ENABLED_SERVICE_APIS", true,
```

Comment:
Should we rename the env varialbe as well ?

Reply:
Probably not a bad idea to do it now as its already a discrete migration to the new api. Its at least opt-out now so the flag is hopefully very uncommon. will send an update